PowerEditor/src/MISC/Common/Common.h

```diff
@@ -45,25 +46,29 @@ const bool dirDown = false;
 #define BCKGRD_COLOR (RGB(255,102,102))
 #define TXT_COLOR    (RGB(255,255,255))
 
-#define generic_strtol wcstol
-#define generic_strncpy wcsncpy
-#define generic_stricmp wcsicmp
-#define generic_strncmp wcsncmp
-#define generic_strnicmp wcsnicmp
-#define generic_strncat wcsncat
-#define generic_strchr wcschr
-#define generic_atoi _wtoi
-#define generic_itoa _itow
-#define generic_atof _wtof
-#define generic_strtok wcstok
-#define generic_strftime wcsftime
-#define generic_fprintf fwprintf
-#define generic_sprintf swprintf
-#define generic_sscanf swscanf
-#define generic_fopen _wfopen
-#define generic_fgets fgetws
-#define generic_stat _wstat
+#define generic_strtol _tcstol
+#define generic_strncpy _tcsncpy
+#define generic_stricmp _tcsicmp
+#define generic_strncmp _tcsncmp
+#define generic_strnicmp _tcsnicmp
+#define generic_strncat _tcsncat
+#define generic_strchr _tcschr
+#define generic_atoi _ttoi
+#define generic_itoa _itot
+#define generic_atof _ttof
+#define generic_strtok _tcstok
+#define generic_strftime _tcsftime
+#define generic_fprintf _ftprintf
+#define generic_sprintf _stprintf
+#define generic_sscanf _stscanf
+#define generic_fopen _tfopen
+#define generic_fgets _fgetts
+#define generic_stat _tstat
+#ifdef UNICODE

```

Comment:
non UNICODE support was dropped some time ago, why should it come back?

, see Revision: 5c48259aee9686921e245f938fe55aebc8951089
Author: Don Ho <don.h@free.fr>
Date: 26.12.2015 01:17:01
Message:
Clean up


Remove unnecessary directive.
----
Modified: PowerEditor/src/MISC/Common/Common.h